pkg/pgn/parser.go

```diff
@@ -15,6 +16,43 @@ import (
 	"github.com/inhies/go-bytesize"
 )
 
+func PathParser(pgn_path string) ([]string, error) {
```

Comment:
functions/methods should be named with verbs or phrases with verbs: func ParsePath() it states what it does
nouns are for variables/types type PathParser  states what it is.